descope/management/user.py

```diff
@@ -498,6 +498,63 @@ def update_display_name(
         )
         return response.json()
 
+    def update_picture(
+        self,
+        login_id: str,
+        picture: str = None,
```

Comment:
I think this argument should not have a default value as if it will be stay None it means this function doing nothing..

Reply:
Having it `None` will delete any existing picture